lib/taurus/qt/qtgui/base/taurusbase.py

```diff
@@ -66,6 +68,23 @@
 DefaultNoneValue = "-----"
 
 
+
+# ------------------------------------------------------------------------
+# Note: this is an experimental feature introduced in v 4.6.4a
+# It may be removed or changed in future releases
+
+scheme_formatters = {}
+for __p in pkg_resources.iter_entry_points('taurus.qt.qtgui.base.formatter'):
+    try:
+        scheme_formatters[__p.name] = '{}.{}'.format(__p.module_name,
+                                                     __p.attrs[0])
+    except Exception as e:
+        warning('Could not load formatter plugin "%s". Reason: %s',
+                __p.module_name, e)
+
+# ------------------------------------------------------------------------
+
```

Comment:
In terms of implementation, I think that this is not the best way of collecting the registered formatters for the following reasons:
- I would call the entry point `taurus.qt.qtgui.base.formatters` (note the final "s"), or even `taurus.qt.formatters`
- The iteration and collection can be simplified (see proposed snippet below)
- The discovery is better delayed until the moment is needed (e.g. define getter for the formatters instead of loading them when importing the module).

In summary, I would do something more or less like this:
```python
def get_registered_formatters():
    formatters = {}
    for ep in pkg_resources.iter_entry_points('taurus.qt.qtgui.base.formatters'):
        try:
            formatters[ep.name] = ep.load()
        except Exception as e:
            taurus.warning('Cannot load "%s" formatter. Reason: %r', ep.name, e)
    return formatters
```

Reply:
Note the getter could also be implemented as a static method of TaurusBaseComponent (to make it available to any widget